README.md

```diff
@@ -25,3 +25,22 @@ B) The generated code itself may change between minor releases. This
 [2]: https://symfony.com/doc/current/setup/flex.html
 [3]: https://symfony.com/doc/current/bundles/SymfonyMakerBundle/index.html
 [4]: https://symfony.com/doc/current/contributing/code/bc.html
+
+## Linting Generated Templates
+
+### PHP-CS-FIXER
+
+MakerBundle uses php-cs-fixer to enforce coding standards when generating `.php`
+files. If you have `friendsofphp/php-cs-fixer` added to your project, we'll
+use the `bin/php-cs-fixer` binary and `.php-cs-fixer.dist.php` configuration file
+automatically.
+
+Otherwise, we use the `src/Bin/php-cs-fixer-v3.13.0.phar` and
```

Comment:
```suggestion
Otherwise, we use a `php-cs-fixer` version that is packaged with this bundle.

No need to mention the version.

Also `php-cs-fixer` probably shouldn't be in `src/Bin` but just maybe `bin/` in our repo :). Or maybe `src/Resources/... something`
```

Reply:
moved to `Resources/bin`